integration/patch_test.go

```diff
@@ -44,14 +48,33 @@ func TestPatch(t *testing.T) {
 
 	for _, img := range images {
 		img := img
+
+		// Only the buildkit instance running within the docker daemon can work
```

Comment:
can we add this to docs please?

Reply:
- [ ] Add to docs that copa only works with standard docker if using the containerd snapshotter.

Reply:
Done